lokalise-file-push/src/main.ts

```diff
@@ -12,18 +12,75 @@ const locales = ghCore.getInput("locales");
 const uploadOptions = ghCore.getInput("upload-options");
 const prDescription = ghCore.getInput("pr-description");
 const keysDescription = ghCore.getInput("keys-description");
+const shouldStaleUnusedKeys = ghCore.getInput("stale-unused");
 
 const LANG_ISO_PLACEHOLDER = "%LANG_ISO%";
 const DEFAULT_PLATFORM = "web";
 
+async function getMainKeys(
+  lokalise: LokaliseApi,
+  page: number
+): Promise<{ id: string; name: string; tags: string[] }[]> {
+  const keys = await lokalise.keys.list({
+    project_id: projectId,
+    filter_tags: ["main"],
+    limit: 500,
+    page,
+  });
+  if (keys && keys.length > 0) {
+    return keys.map((key) => ({
+      id: key.key_id,
+      name: key["key_name"]["web"],
+      tags: key.tags,
+    }));
+  }
+  return [];
+}
+
+async function getAllMainKeys(
+  lokalise: LokaliseApi
+): Promise<{ id: string; name: string; tags: string[] }[]> {
+  let all: any = [];
+  let page = 0;
+  let mainKeysPerPage = [];
+  while (page === 0 || mainKeysPerPage.length === 500) {
+    page++;
+    mainKeysPerPage = await getMainKeys(lokalise, page);
+    all = [...all, ...mainKeysPerPage];
+  }
+  return all;
+}
+
+async function staleUnusedKeys(lokalise: LokaliseApi, file: any) {
```

Comment:
Do we have a type for file? 
```suggestion
async function staleUnusedKeys(lokalise: LokaliseApi, file: any) {
```